### PR TITLE
#2201 enriched InvestWithRegularEther with test for portfolio section.

### PR DESCRIPTION
**Pull request by Desperado**
updated `InvestWithRegularEther` with verifications on portfolio section

### CODE REVIEW

File: app/components/modals/tx-sender/investment-flow/Investment.tsx [hunk 1]
```diff
@@ -277,8 +278,10 @@ export const InvestmentSelectionComponent: React.FunctionComponent<IProps> = ({
               <Label>
                 <FormattedMessage id="investment-flow.estimated-neu-tokens" />
               </Label>
-              <InfoAlert>
-                {(showTokens && neuReward && `${formatEurTsd(neuReward)} NEU`) || "\xA0"}
+              <InfoAlert data-test-id="invest-modal.est-neu-tokens">
+                {(showTokens &&
+                  neuReward && <Money value={neuReward} currency={ECurrency.NEU} />) ||
```
Comment (desfero) [inline app/components/modals/tx-sender/investment-flow/Investment.tsx]: `formatEurTsd` is incorrect usage here as tokens should always have 4 decimals

**Comment by Desperado** [comment]
@Moejoe90 could you please review?

File: app/components/modals/tx-sender/investment-flow/Investment.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/modals/tx-sender/investment-flow/Investment.tsx]: good catch

File: app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts [hunk 2]
```diff
@@ -1,29 +1,57 @@
...
+  assertButtonIsActive,
+  assertDashboard,
+  confirmAccessModal,
+  etoFixtureAddressByName,
+  goToDashboard,
+} from "../utils";
 import { tid } from "../utils/selectors";
 import { createAndLoginNewUser } from "../utils/userHelpers";
 
+const parseAmount = (amount: string) => parseFloat(amount.replace(" ", ""));
```
Comment (Moejoe90) [inline app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts]: are you sure the numbers are always gonna get parsed `parseFloat` wouldn't be better to use `BigNumber` for comparison

Comment (desfero) [reply app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts]: make sense, will fix in PR for accept payout e2e test

**Pull request merged**